kpi/views/environment.py

```diff
@@ -31,7 +31,10 @@ class EnvironmentView(APIView):
             'SECTOR_CHOICES',
             # Intentional t() call on dynamic string because the default
             # choices are translated (see static_lists.py)
-            # Strip \r for legacy reasons
+            # \n vs \r\n - In django-constance <2.7.0, new lines were saved as "\r\n"
+            # Starting in 2.8, new lines are saved as just "\n". In order to ensure compatibility
```

Comment:
Actually, it's starting in 2.7 as you said just the line above ;-) but not a big deal.